src/nvim/options.lua

```diff
@@ -940,6 +940,14 @@ return {
       redraw={'current_window'},
       defaults={if_true="foldtext()"}
     },
+    {
+      full_name='foldlevelmarker', abbreviation='fdlm',
+      short_desc=N_("Enable digits shown to indicate the nesting level"),
+      type='bool', scope={'global'},
```

Comment:
Shouldn't this be `window` scoped like other fold options?

Reply:
That's how I implemented this at first (and which I can revert to), after some testing and contemplation I decided to go this way because because it feels like it should be a global setting since it's related to the gutter visuals.

Reply:
You're probably right here. So I reverted back to window scoped.

Reply:
Yeah was gonna say `'signcolumn'` and `'number'` are window scoped too.